test/WebJobs.Script.Tests/CSharpEndToEndTests.cs

```diff
@@ -32,6 +32,25 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         }
 
         [Fact]
+        public async Task NotificationHub()
+        {
+            await NotificationHubTest();
+        }
+
+        [Fact]
+        public async Task NotificationHub_Out_Notification()
+        {
```

Comment:
@brettsam currently, if the host is not running, the project.json will **not** be processed. This was a limitation we were aware of with the current design but have plans to address. @mattchenderson had that requirement for something he is working on and we validated that, as long as the host is running, the packages are restored and loaded as expected.
There are some things we can do to simplify the tests so you don't have to worry about the project.json, we can revisit this.